2024/json-ld-wg/index.html

```diff
@@ -55,7 +55,7 @@
           <li><a href="#background">Background</a></li>
           <li><a href="#scope">Scope</a></li>
           <li><a href="#deliverables">Deliverables</a></li>
-		  <li><a href="#success-criteria">Success Criteria</a></li>
+		      <li><a href="#success-criteria">Success Criteria</a></li>
```

Comment:
I assume this doesn't need indenting?
```suggestion
          <li><a href="#success-criteria">Success Criteria</a></li>
```

Reply:
Probably ignore this suggestion.  There is a tab vs space indent battle in this file.  This is just one problematic line. 
 Could clean the whole file in another PR if there is interest.